### PR TITLE
Fix closer zooms in LightFX.cpp

### PR DESCRIPTION
**Pull request by mrmbernardi**

### CODE REVIEW

File: src/openrct2/drawing/LightFX.cpp [hunk 1]
```diff
@@ -386,8 +382,8 @@ void LightFXPrepareLightList()
             entry->LightIntensity = static_cast<uint8_t>(
                 std::min<uint32_t>(0xFF, (entry->LightIntensity * lightIntensityOccluded) / (totalSamplePoints * 100)));
         }
-        entry->LightIntensity = static_cast<uint8_t>(
-            std::max<uint32_t>(0x00, entry->LightIntensity - static_cast<int8_t>(_current_view_zoom_front) * 5));
+        entry->LightIntensity = static_cast<uint8_t>(std::max<uint32_t>(
+            0x00, entry->LightIntensity - std::max<int8_t>(0, static_cast<int8_t>(_current_view_zoom_front)) * 5));
```
Comment (Gymnasiast) [inline src/openrct2/drawing/LightFX.cpp:386]: This is _extremely_ hard to parse. Can we break this up a bit?

Comment (mrmbernardi) [reply src/openrct2/drawing/LightFX.cpp]: I've rewritten it

**Review by Gymnasiast — APPROVED** [review]
